.github/templates/build-container/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: build
```

Comment:
why not `.github/actions/*/action.yaml` ?

Reply:
`templates` sounded more appropriate - we are going to use the actions as templates that can be used in other actions.

Reply:
IMO since what u create is an action that u use throughout the workflows it makes sense it's under actions folder,
`.github/templates/build-container/action.yaml` is more of an action than a template

Reply:
ok, as you like, I can rename :+1: